### PR TITLE
Improve logging when archiving files

### PR DESCRIPTION
**Pull request by sschuberth**
Only log (the performance) in the downstream `archive()` function. The
package coordinates are still visible from the logged directory name.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: scanner/src/main/kotlin/PathScanner.kt [hunk 1]
```diff
@@ -138,7 +138,10 @@ abstract class PathScanner(
 
         remainingPackages.removeAll { it in resultsFromStorage.keys }
 
-        log.info { "Scanning ${remainingPackages.size} package(s) for which no stored scan results were found." }
+        log.info {
+            "Scanning ${remainingPackages.size} package(s) for which no stored scan results were found out of total " +
+                    "${packages.size} package(s)."
```
Comment (mnonnenmacher) [inline scanner/src/main/kotlin/PathScanner.kt:143]: "... out of **a total of** x package(s)."

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: scanner/src/main/kotlin/PathScanner.kt (hunk 1 above)
Comment (mnonnenmacher) [reply scanner/src/main/kotlin/PathScanner.kt:143]: The sentence sounds a bit clumsy, how about changing the log above instead to "Found stored scan results for x of y packages matching $scannerCriteria", which would also be more correct, as technically `resultsFromStorage.size` is the number of packages for which results were found, not the number of scan results in the map.

Comment (sschuberth) [reply scanner/src/main/kotlin/PathScanner.kt:143]: Wuold you then remove the

    log.info { "Found ${resultsFromStorage.size} stored scan result(s) matching $scannerCriteria." }

from a few lines above?

**Review by mnonnenmacher — APPROVED** [review]

File: scanner/src/main/kotlin/PathScanner.kt (hunk 1 above)
Comment (mnonnenmacher) [reply scanner/src/main/kotlin/PathScanner.kt]: Resolved offline.

**Pull request merged**